freegames/hangman.py

```diff
@@ -0,0 +1,107 @@
+"""hangman, letter guessing game
+
+Exercises
+1. Change Hangman's picture (position, length, etc)
+2. Make a additional dictionary, not a fruit
+3. Convert lowercase letters to uppercase letters
+4. Make it possible to enter not only the alphabet, but also the word to guess the answer
+"""
+
+from turtle import *
+from random import choice
+
+def screen_write(letter):
+    """write text on the screen"""
+    write(letter, move=True, font=("Courier", 14, "normal"))
+
+def move_write(x, y, letter):
+    """After move, write text on the screen"""
+    goto(x, y)
+    screen_write(letter)
+
+
+def draw(x, y, angle, length, straight):
+    """draw a hangman game"""
+    goto(x, y)
+    pendown()
+    left(angle)
+    if straight == True:
+        forward(length)
+    elif straight == False:
+        circle(length, None, 100)
+    penup()
+
+def drawHangman(num):
+    if num == 1: 
+        """draw head"""
```

Comment:
Use comments, not strings like `# Draw head`. Same for other strings in this function.